### PR TITLE
Checkpoint 5

### PR DESCRIPTION
**Pull request by DarylMcCullough**
I have added the ability to view events from the show view for the registered app. Also, I now have the Heroku deployment working (the problem was that my code for setting up a default url was in environments/development, but not environments/production.) See the [Heroku version](https://ez-events.herokuapp.com). (But since seeds.rb doesn't run in production, there are no events to see.)

### CODE REVIEW

File: spec/controllers/registered_applications_controller_spec.rb [hunk 1]
```diff
@@ -35,6 +35,8 @@
         get :show, params: {id: my_app.id}
         expect(assigns(:registered_application)).to eq(my_app)
       end
+
```
Comment (brittanyjspringer) [inline spec/controllers/registered_applications_controller_spec.rb:38]: @DarylMcCullough what is this? Please remove.

File: db/seeds.rb [hunk 2]
```diff
@@ -1,7 +1,53 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+    require 'faker'
```
Comment (brittanyjspringer) [inline db/seeds.rb:1]: I cloned down the repo, dropped the database and tried to seed it. 

I got: ActiveRecord::RecordInvalid: Validation failed: Name is too short (minimum is 5 characters)

**Review by brittanyjspringer — CHANGES_REQUESTED** [review]

**Comment by brittanyjspringer** [comment]
@DarylMcCullough once this is all resolved, please reassign back to me. Thanks!

File: spec/controllers/registered_applications_controller_spec.rb (hunk 1 above)
Comment (DarylMcCullough) [reply spec/controllers/registered_applications_controller_spec.rb:38]: I put that in by analogy from the Bloccit Checkpoint 19 spec for PostsController.show.

```
   describe "GET show" do
     it "returns http success" do
 # #16
       get :show, {id: my_post.id}
       expect(response).to have_http_status(:success)
     end
     it "renders the #show view" do
 # #17
       get :show, {id: my_post.id}
       expect(response).to render_template :show
     end
 
     it "assigns my_post to @post" do
       get :show, {id: my_post.id}
 # #18
       expect(assigns(:post)).to eq(my_post)
     end
   end
```

File: db/seeds.rb (hunk 2 above)
Comment (DarylMcCullough) [reply db/seeds.rb:1]: Not sure how to force Faker to use longer names. I'm a little surprised that Faker would produce such a short name. Maybe there is something else going on? Anyway, assuming that the problem is with Faker sometimes generating too-short names, I put a loop in the seeds.db that keeps generating names until it gets one with more than 5 characters.

File: spec/controllers/registered_applications_controller_spec.rb (hunk 1 above)
Comment (DarylMcCullough) [reply spec/controllers/registered_applications_controller_spec.rb:38]: The point of that line is that in the RegisteredApplicationsController, there is a line:

```
def show
         @registered_application = RegisteredApplication.find(params[:id])
...
```

This line is tested by writing
```
expect(assigns(:registered_application)).to eq(my_app)
```

where `my_app` is the app being "shown".

**Comment by brittanyjspringer** [comment]
Tests pass and seeds work. Thanks!

**Review by brittanyjspringer — APPROVED** [review]

**Pull request merged**